packages/mds-stream/nats-streaming/nats.ts

```diff
@@ -8,7 +7,7 @@ export type EventProcessor<TData, TResult> = (type: string, data: TData) => Prom
 const SUBSCRIPTION_TYPES = ['event', 'telemetry'] as const
 type SUBSCRIPTION_TYPE = typeof SUBSCRIPTION_TYPES[number]
 
-const subscriptionCb = async <TData, TResult>(processor: EventProcessor<TData, TResult>, msg: stan.Message) => {
+const subscriptionCb = async <TData, TResult>(processor: EventProcessor<TData, TResult>, msg: any) => {
```

Comment:
No type for `msg` parameter?

Reply:
Not yet (hoping to add it as part of this PR) -- I need to actually test the client still. It doesn't seem like NATS.js providers a type for the message; while I think they're a string, gotta double check.